src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/ReferralConcluder.kt

```diff
@@ -64,4 +66,10 @@ class ReferralConcluder(
 
     return null
   }
+
+  private fun getNumberOfAttendedAppointments(referral: Referral): Int {
+    return referral.actionPlan?.let {
+      actionPlanRepository.countNumberOfAttendedSessions(it.id)
```

Comment:
Can we rename this to `countSessionsWithAttendanceRecorded` (or similar), please? "Attended" in domain terminology is when people turn up to their appointment

Reply:
Will do